tests/Unit/Infrastructure/Persistence/IlluminateTalkRepositoryTest.php

```diff
@@ -0,0 +1,52 @@
+<?php
+
+namespace OpenCFP\Test\Unit\Infrastructure\Persistence;
+
+use OpenCFP\Domain\Model;
+use OpenCFP\Domain\Talk;
+use OpenCFP\Infrastructure\Persistence\IlluminateTalkRepository;
+use OpenCFP\Test\Helper\Faker\GeneratorTrait;
+use PHPUnit\Framework;
+
+/**
+ * @covers \OpenCFP\Infrastructure\Persistence\IlluminateTalkRepository
+ */
+final class IlluminateTalkRepositoryTest extends Framework\TestCase
+{
+    use GeneratorTrait;
+
+    public function testImplementsTalkRepository()
+    {
+        $reflection = new \ReflectionClass(IlluminateTalkRepository::class);
+
+        $this->assertTrue($reflection->implementsInterface(Talk\TalkRepository::class));
+    }
+
+    public function testPersistSavesTalk()
+    {
+        $talk = $this->createTalkMock([
+            'save',
+        ]);
+
+        $talk
+            ->expects($this->once())
+            ->method('save');
+
+        $repository = new IlluminateTalkRepository();
+
+        $repository->persist($talk);
```

Comment:
I am not sure what [`IlluminateTalkRepository::persist()`](https://github.com/opencfp/opencfp/blob/70b5aff3fb766b38642a6f2ae3f350871d740fae/classes/Infrastructure/Persistence/IlluminateTalkRepository.php#L17) is supposed to return, as the method `Model\Talk::save()` appears to be a magic method.

Reply:
Yeah, same issue as I indicated in #744